backend/app/users/router.py

```diff
@@ -65,3 +65,16 @@ def mark_tour_as_seen(
     user: User = Depends(get_authenticated_user),
 ) -> None:
     UserService(db).mark_tour_as_seen(user.id)
+
+
+@router.post(
+    "/can_become_admin",
+    dependencies=[
+        Depends(Authorization.enforce(Action.GLOBAL__CREATE_USER, EmptyResolver)),
+    ],
+)
+def can_become_admin(
```

Comment:
I don't like the name of this function, the wording is phrased like a question. 
Maybe change it to, one of these two:

```suggestion
def configure_can_become_admin(
def set_can_become_admin(
```